raiden/constants.py

```diff
@@ -57,6 +58,7 @@
 ZERO_TOKENS = TokenAmount(0)
 
 ABSENT_SECRET = Secret(b"")
+ABSENT_BLOCKTIMEOUT = BlockTimeout(0)
```

Comment:
Could you please revert this? A block timeout of `0` is most likely invalid. For the case of a lock timeout it will always create an invalid lock expiration, if we do instantiate this value the type checker cannot help us find bugs.